### PR TITLE
Notification when borrowing is created

### PR DESCRIPTION
**Pull request by Sergi0bbb**

### CODE REVIEW

File: borrowing/views.py [hunk 1]
```diff
@@ -26,3 +30,15 @@ def get_serializer_class(self):
...
         elif self.action == "retrieve":
             return BorrowingDetailSerializer
+        return BorrowingSerializer
+
+    def create(self, request, *args, **kwargs):
+        response = super().create(request, *args, **kwargs)
+
+        if response.status_code == status.HTTP_201_CREATED:
+            borrowing = response.data
+            message = f"New borrowing created: {borrowing["id"]}"
```
Comment (pashawarganov) [inline borrowing/views.py]: make more informative message like "id, book, expected return date"

**Review by olena4reunited — CHANGES_REQUESTED** [review]
Move all changes in file notification_when_borrowing_created.py to new file outside the apps with descriptive name telegram_bot.py

File: borrowing/notification_when_borrowing_created.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+from aiogram import Bot, Dispatcher, Router
+from aiogram.client.default import DefaultBotProperties
+from aiogram.enums import ParseMode
+from dotenv import load_dotenv
+
+load_dotenv()
+TOKEN = getenv("BOT_TOKEN")
+CHAT_ID = getenv("CHAT_ID")
+bot = Bot(token=TOKEN, default=DefaultBotProperties(parse_mode=ParseMode.HTML))
```
Comment (yashkunn) [inline borrowing/notification_when_borrowing_created.py]: I recommend adding a check like
if not TOKEN or not CHAT_ID:
    ...

File: .env.sample [hunk 3]
```diff
@@ -0,0 +1,2 @@
+BOT_TOKEN=YOUR_BOT_TOKEN
+CHAT_ID=YOUR_TELEGRAM_CHAT_ID
```
Comment (pashawarganov) [inline .env.sample]: add empty line in end of file

**Review by pashawarganov — CHANGES_REQUESTED** [review]
Make some changes

**Review by ABaidin — APPROVED** [review]

**Review by panichqa — APPROVED** [review]

**Review by panichqa — APPROVED** [review]

**Review by yashkunn — APPROVED** [review]

**Review by yashkunn — APPROVED** [review]

**Review by yashkunn — APPROVED** [review]

**Review by ABaidin — APPROVED** [review]

**Review by olena4reunited — APPROVED** [review]

**Review by pashawarganov — APPROVED** [review]

**Pull request merged**